ui/app/mainui/AppMain.qml

```diff
@@ -75,6 +75,7 @@ Item {
     readonly property TransactionStore transactionStore: TransactionStore{
         walletAssetStore: appMain.walletAssetsStore
     }
+    property CurrenciesStore currencyStore: CurrenciesStore{}
```

Comment:
The `property var rootChatStore: ChatStores.RootStore` above on line 61 already has a `currencyStore` defined; maybe it's worth to pass it down there to share the same instance?